x-pack/plugins/case/common/api/cases/status.ts

```diff
@@ -7,6 +7,8 @@
 
 import * as rt from 'io-ts';
 
+export const AllCaseType = 'all';
```

Comment:
This can be `export const AllCaseType = 'all' as const;`. This will help you avoid doing stuff like `x = 'all' as AllCaseType `